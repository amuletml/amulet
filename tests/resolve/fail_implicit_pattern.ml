let implicit (a, b) = (1, 2)
