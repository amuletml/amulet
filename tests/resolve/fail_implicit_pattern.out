let implicit#0 (a#1, b#2) = (1, 2)
